src/ontology/mondo-edit.obo

```diff
@@ -83836,6 +83836,7 @@ xref: UMLS:C1515284 {source="MONDO:equivalentTo", source="MONDO:MEDGEN", source=
 is_a: MONDO:0003395 {source="DOID:8394", source="NCIT:C39946"} ! testicular granulosa cell tumor
 intersection_of: MONDO:0003395 ! testicular granulosa cell tumor
 intersection_of: has_characteristic HP:0003581 ! Adult onset
+relationship: excluded_from_qc_check http://purl.obolibrary.org/obo/mondo/sparql/qc/general/qc-single-child.sparql
```

Comment:
@twhetzel - this single child excluded_from_qc_check is not necessary since this term is not a single child. But it is not "hurting" anything. 
At this point, I think it is ok to keep it as is so we can merge this PR. 
Let's remove it in a separate PR though.

Reply:
Ok, in the first run of the qc checks this was an issue. As I look at it again, this was the sign of bigger problems to come (the logical issue you resolved).